config/crd/bases/argoproj.io_notificationsconfigurations.yaml

```diff
@@ -1,9 +1,11 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.6.1
```

Comment:
You might need to update your local controller-gen version. During operator-sdk upgrade, it is bumped up to 0.14.0.